rdkit/Chem/UnitTestDescriptors.py

```diff
@@ -50,6 +50,9 @@ def testBadAtomHandling(self):
       for nm, fn in Descriptors._descList:
         try:
           v = fn(m)
+        except RuntimeError:
+          # 3D descriptors fail since the mol has no conformers
+          pass
```

Comment:
I need to make the test if I want to correct my code. Want should I do to be able to compute them.